.github/workflows/grumpy-reviewer.lock.yml

```
@@ -1044,15 +1004,19 @@ jobs:
+      needs.activation.outputs.oauth_token_check_failed == 'true' || needs.activation.outputs.stale_lock_file_failed == 'true' ||
+      needs.activation.outputs.secret_verification_result == 'failed' || needs.activation.outputs.daily_ai_credits_exceeded == 'true')
     runs-on: ubuntu-slim
     permissions:
-      contents: read
+      actions: read
+      issues: write
       pull-requests: write
     concurrency:
       group: "gh-aw-conclusion-grumpy-reviewer"
       cancel-in-progress: false
       queue: max
```

actionlint (1.7.12)

[error] unexpected key "queue" for "concurrency" section. expected one of "cancel-in-progress", "group"

(syntax-check)